### ISSUE TITLE
Constant "update ready" status when run on unRAID server

### ISSUE THREAD

**Issue by dlandon** [tokyis/docker-zoneminder#1]
I installed the docker on unRAID and it shows that an update is ready and when I click on the update ready it updates nothing and then shows update ready again.  I noticed you forked from aptalca's zoneminder and I wonder if there is a file missing that indicates an update.

**Comment by tokyis** [comment]
Yes I think it's because I'm not using the master branch in the template. I have just been ignoring it since I initially forked from aptalca for my own tweaks.
